chains/celo/listener.go

```diff
@@ -33,3 +44,28 @@ func (l *listener) start() error {
 func (l *listener) close() {
 	l.conn.Close()
 }
+
+func (l *listener) getTransactionBlockHash(hash common.Hash) (blockHash common.Hash, err error) {
+
+	receipt, err := l.conn.Client().TransactionReceipt(context.Background(), hash)
+	if err != nil {
+		return hash, fmt.Errorf("unable to get BlockHash: %s", err)
+	}
+	return receipt.BlockHash, nil
+}
+
+func (l *listener) getBlockTransactionsByHash(hash common.Hash) (txHashes []common.Hash, txRoot common.Hash, err error) {
+	block, err := l.conn.Client().BlockByHash(context.Background(), hash)
+	if err != nil {
+		return []common.Hash{}, common.Hash{}, fmt.Errorf("unable to get BlockHash: %s", err)
```

Comment:
just return nil, nil, and the error here

Reply:
Can't use nil for the second parameter or else I get an error 
`Cannot use nil for type common.Hash`